src/blog/testrpc-is-now-ganache.md

```diff
@@ -24,8 +24,8 @@ Additionally, we realized over time that the TestRPC was a poor name. The TestRP
 
 With this name change, we also released [a fully-interactive, graphical version of Ganache](/ganache).
 
-The command-line utility is still available via NPM; the [command line flags](https://github.com/trufflesuite/ganache-cli/blob/master/README.md) are the same too. All that's changed is the name of the command.
+The command-line utility is still available via NPM; the [command line flags](https://github.com/trufflesuite/ganache-cli-archive/blob/master/README.md) are the same too. All that's changed is the name of the command.
```

Comment:
maybe this should be https://github.com/trufflesuite/ganache-cli-archive/blob/master/README.md#options

By the way, feel free to ignore some of these comments about specific URLs in the interest of getting the initial, "more important work" through.

Reply:
These will have to change again the future to point to actual ganache documentation instead of these readmes, so I don't think it is worth changing now.